src/EditorFeatures/Core/ExtractMethod/AbstractExtractMethodCommandHandler.cs

```diff
@@ -151,10 +151,18 @@ public bool ExecuteCommand(ExtractMethodCommandArgs args, CommandExecutionContex
 
             // apply the change to buffer
             // get method name token
-            ApplyChangesToBuffer(result, textBuffer, cleanupOptions, cancellationToken);
+            SyntaxToken methodNameAtInvocation;
```

Comment:
Do we ever expect this to be a default value? From code it is hard to tell

Reply:
no.  it can't ever be a default value.  NOte: this is not assigned anything at the decl point.  It's purely a declaration of the variable.  there is one and only one value assignment to it from the method invoked.

Reply:
oh yea, I meant is it guaranteed to always be _something_ or do we have cases where the returned value is wrong? Not blocking on this PR, just curious

Reply:
We weren't using it before, so I could see that this might be an area for a bug we don't know about :) Don't spend time on it, just if you know off hand.